openedx/core/djangoapps/cors_csrf/decorators.py

```diff
@@ -22,6 +23,24 @@ def _inner(*args, **kwargs):  # pylint: disable=missing-docstring
             # the cross-domain version of the CSRF cookie.
             request = args[0]
             request.META['CROSS_DOMAIN_CSRF_COOKIE_USED'] = True
+            current_authenticator = getattr(request, 'successful_authenticator', None)
+
+            if hasattr(request, "resolver_match"):
```

Comment:
Would it be too conservative to have a broad try/except with? Same question anywhere below where we are using an attribute or function.
```
set_custom_attribute("tmp_cors_csrf_decorator.error", str(e))
```